svg-icons-builder/test-helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonConversionOptions } from 'svg-to-ts';
+import { Delimiter } from 'svg-to-ts/src/lib/generators/code-snippet-generators';
```

Comment:
This is also something we should export from `svg-to-ts`. I will do this and then adjust later once a new version from `svg-to-ts` is deployed.

Reply:
Sounds good!